resources/commands.json

```diff
@@ -1396,6 +1421,34 @@
 						"description": "The NFT type"
 					}
 				]
+			}, {
+				"name": "balance [nftAddress] [did]",
+				"description": "It gets the balance of a NFT",
+				"details": "This command allows to check if an address is a holder of a specific NFT wrapped by a Nevermined DID",
+				"examples": ["ncli nfts1155 hold did:nv:afd733c23c41af948be7ec039c3fb2048d437e082a69ea3f336cdf452a49be7e"],
```

Comment:
this example does not match with the command `balance` vs `nfts 1155 hold`

Reply:
fixed